src/main/java/games/strategy/engine/lobby/server/db/DbUserController.java

```diff
@@ -67,6 +80,12 @@ public void updateUser(final DBUser user, final HashedPassword password) {
     secondaryDao.updateUser(user, password);
   }
 
+  @Override
+  public void updateUser(final DBUser user, final String password) {
```

Comment:
Sorry restarting, do the correct encryption before we hit these methods, they probably should not change much at all.